### DISCUSSION TITLE
What common tools for reading structured logs?

### DISCUSSION THREAD

**Discussion by flandia** [uber-go/zap#1293] [Q&A]
Hi, I am a new developer to Golang. In development, I found out that many famous Golang logging libraries use "structured logging" as their default output format. It said that "structured logs" are easy to parse, and make debugging much easier. However, I didn't find it useful, since I had to humanly parse JSON format on the terminal of the IDE, or on the SSH console.

In my imagination, there should be convenient tools for parsing those logs, maybe similar to MS Excel that can filter or sort by any field. I haven't found such tools on Google (searching "structured logging tools") and on plugin marketplace of Goland and VS Code.

So, in conclusion, what tools do you often use to read structured logs?
